accessibility-checker-engine/src/v2/checker/accessibility/nls/index.ts

```diff
@@ -494,13 +494,13 @@ let a11yNls = {
     "RPT_Table_DataHeadingsAria": {
         0: "Data table must identify headers",
         "Pass_0": "Rule Passed",
-        "Fail_1": "Data table has no headers identified"
+        "Fail_1": "Table has no headers identified"
     },
     // JCH - DONE
     "WCAG20_Label_RefValid": {
         0: "The 'for' attribute must reference a non-empty, unique 'id' attribute of an <input> element",
         "Pass_0": "Rule Passed",
-        "Fail_1": "The <label> 'for' attribute value of \"{0}\" is not the 'id' of a valid <input> element"
+        "Fail_1": "The value \"{0}\" of the `<label>` `for` attribute is not the `id` of a valid `<input>` element"
```

Comment:
We can't have back ticks in the error messages, so the Fail_1 condition here has to be:

"Fail_1": "The value \"{0}\" of the <label> 'for' attribute is not the 'id' of a valid <input> element"

Reply:
Done, thanks